spec/03-plugins/26-prometheus/02-access_spec.lua

```diff
@@ -611,3 +613,226 @@ describe("Plugin: prometheus (access) granular metrics switch", function()
 
 end)
 end
+
+describe("Plugin: prometheus (access) AI metrics", function()
```

Comment:
let's also add a test that by default ai metrics are not displayed. you can append on in the previous `describe` block.

Reply:
Will work on it now, thanks